### PR TITLE
feat: fronend.js optimization

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Optimization of fronend.js

### Types of changes
Improvements.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests

### CODE REVIEW

File: assets/js/unminified/frontend.js [hunk 1]
```diff
@@ -622,7 +626,7 @@ var main_header_masthead = document.getElementById('masthead');
 	    	M.splice(1,1,tem[1]);
 	    }
 
-	    bodyElement = document.body;
+	    bodyElement = body;
```
Comment (premanshup) [inline assets/js/unminified/frontend.js:629]: @sagarb90 If body is a global variable then we can remove  `bodyElement = body;` this assignment and use `body` directly.

File: assets/js/unminified/frontend.js [hunk 2]
```diff
@@ -820,7 +824,7 @@ var main_header_masthead = document.getElementById('masthead');
 
 	/* Add class if mouse clicked and remove if tab pressed */
 	if ( 'querySelector' in document && 'addEventListener' in window ) {
-		var body = document.body;
+		var body = body;
```
Comment (premanshup) [inline assets/js/unminified/frontend.js:827]: @sagarb90 If body is a global variable then we can remove  `var body = body;` this assignment and use `body` directly.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: assets/js/unminified/frontend.js (hunk 2 above)
Comment (sagarb90) [reply assets/js/unminified/frontend.js]: @premanshup Done mentioned changes.

File: assets/js/unminified/frontend.js (hunk 1 above)
Comment (sagarb90) [reply assets/js/unminified/frontend.js]: @premanshup Done mentioned changes.

**Review by premanshup — APPROVED** [review]

**Review by shubhw12 — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**